src/openzaak/conf/includes/base.py

```diff
@@ -290,12 +290,11 @@
 # Django setup configuration
 #
 SETUP_CONFIGURATION_STEPS = [
-    "openzaak.config.bootstrap.site.SiteConfigurationStep",
-    "openzaak.config.bootstrap.notifications.AuthNotificationStep",
-    "openzaak.config.bootstrap.notifications.NotificationsAPIConfigurationStep",
-    "openzaak.config.bootstrap.selectielijst.SelectielijstAPIConfigurationStep",
-    "openzaak.config.bootstrap.demo.DemoUserStep",
-    "openzaak.config.bootstrap.authorizations.AuthorizationConfigurationStep",
+    "django_setup_configuration.config.sites.steps.SitesConfigurationStep",
```

Comment:
The command `python src/manage.py setup_configuration --yaml-file docker/setup_configuration/data.yml` currently crashes: 

```
CommandError: Unable to import step django_setup_configuration.config
```

```suggestion
    "django_setup_configuration.contrib.sites.steps.SitesConfigurationStep",
```

Also `zgw-consumers` has to be upgraded to >=0.36.0 to be able to use the `ServiceConfigurationStep`

Reply:
I thought we had a test but I remember that I deleted it.

Maybe an action that runs `python src/manage.py setup_configuration --yaml-file docker/setup_configuration/data.yml` would be useful

Reply:
> Maybe an action that runs python src/manage.py setup_configuration --yaml-file docker/setup_configuration/data.yml would be useful

I think as a sanity check that would be good, maybe it could be similar to the `quickstart` job in objects: https://github.com/maykinmedia/objects-api/blob/master/.github/workflows/quick-start.yml

Reply:
created a separate issue for this https://github.com/maykinmedia/open-api-framework/issues/96